tests/models/test_getmodel_unfusestrategy.py

```diff
@@ -6,25 +6,23 @@
 from fms.models import get_model
 
 
-# How to set unfuse_strategy to obtain desired `target_model` fusion
+# How to set fused_weights to obtain desired `target_model` fusion
 #
-# ckpt       target_model   unfuse_strategy   tested
+# ckpt       target_model   fused_weights     tested
 #                            FP16    GPTQ
 # --------------------------------------------------
-# none       fused           None    None     (Y, Y)
-# none       unfused         post    pre      (Y, Y)
```

Comment:
the tests with `model_path=None` still exist, for both FP16 and GPTQ.
The first two line of this table should be:
```
none       fused           True    True       (Y, Y)
none       unfused         False   False      (Y, Y)
```